# PANAM2-ENTERO

This repository contains a dedicated database for analysing enterovirus with PANAM2. For using the pipelines, read in a first step the README associated to the 
repository PANAM2 and clone it on your computer (https://github.com/panammeb/PANAM2.git). 

In a second step clone this repository and the file ".ini" (entero.ini as example included) must include at least the following lines:

REFERENCE_BASE	<path to>/bd_entero

The primers targetting the VP1 capsid are the following:

FORWARD_PRIMER_SEQUENCE	ACCAGCACTGACAGCAGYNGARAYNGG

REVERSE_PRIMER_SEQUENCE	TACTGGACCACCTGGNGGNAYRWACAT

Examples for filling the ".ini" file can be found in the PANAM2 repository 

PANAM2 combines the publicly available tools: VSEARCH, HMMER, FASTTREE, KRONA, R (with packages Vegan, Phyloseq, Picante, Mass, Cluster) 
with 3 main perl scripts dedicated to:

- step 1 - **quality_panam.pl**: merging, demultiplexing and cleaning raw Illumina reads

- step 2 - **preprocess_panam.pl**: clustering and cleaning Enterovirus according to their abundances (remove rares for example)

- step 3 - **taxonomy_panam.pl**: processing similarity annotations, alignments and phylogenies and computing main diversity indices for each samples and taxonomic units  (system dependencies:bioperl)
 

If you use this package, please cite:

Bisseux M, Debroas D, Mirand A et al. Monitoring of enterovirus diversity in wastewater by ultra-deep sequencing: An effective complementary tool for clinical 
enterovirus surveillance. Water Research 2020;169:115246.

Taib N, Mangot J-F, Domaizon I, Bronner G, Debroas D. (2013). Phylogenetic Affiliation of SSU rRNA Genes Generated by Massively Parallel Sequencing: New Insights 
into the Freshwater Protist Diversity. PLoS ONE 8:e58950.

